test/test_examples_3d.jl

```diff
@@ -190,7 +190,11 @@ const EXAMPLES_DIR = joinpath(pathof(Trixi) |> dirname |> dirname, "examples", "
             l2   = [0.0038729054515012624, 0.00903693761037057, 0.0041729297273898815, 0.01160504558506348, 0.006241548790045999, 0.009227641613254402, 0.0034580608435846143, 0.011684993365513006, 0.0022068452165023645],
             linf = [0.012628629484152443, 0.03265276295369954, 0.012907838374176334, 0.044746702024108326, 0.02796611265824822, 0.03453054781110626, 0.010261557301859958, 0.044762592434299864, 0.010012319622784436])
   end
-  # too expensive for CI
+  @testset "parameters_mortar_alfven_wave.toml" begin
+    test_trixi_run(joinpath(EXAMPLES_DIR, "parameters_mortar_alfven_wave.toml"),
+            l2   = [0.004222069353491928, 0.008615262970183092, 0.004350750049989715, 0.010578512497282935, 0.007396881997755467, 0.008578781812069523, 0.003589437565590415, 0.010672080969050656, 0.002433803112282749],
+            linf = [0.024102636344802253, 0.055214024613181104, 0.02967661132645796, 0.06663480547645528, 0.0454208040875832, 0.051820917119794974, 0.024459143124548333, 0.06552275581822167, 0.01719389063962334])
+  end  # too expensive for CI
```

Comment:
Does it make sense to add also a copy of the OTV (or similar?) test such that the HLL flux in 3D is used properly, or is this not an issue for 3D?

Reply:
I need to think about this. There is a 3D extension of OTV but it is not a standard test.

Reply:
Well, if the coverage does not change because the existing HLL was already not tested properly either, I think we are ok ;-) Let's wait for the report.